Lanchonete/LanchoneteTests/Managers/SandwichManagerTests.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  SandwichManagerTests.swift
+//  LanchoneteTests
+//
+//  Created by vera.dias on 1/26/21.
+//
+
+import XCTest
+@testable import Lanchonete
+
+class SandwichManagerTests: XCTestCase {
+    func test_GIVEN_a_menu_sandwich_WHEN_I_add_ingredient_THEN_the_price_is_updated(){
+        //Given
+        let sandwichManager = SandwichManager(with: .xBacon)
+        
+        //When
+        sandwichManager.addIngredient(.egg)
+        
+        //Then
+        XCTAssertEqual(sandwichManager.getPrice(), 9.0)
+    }
+    
+    func test_GIVEN_a_menu_sandwich_WHEN_I_add_ingredient_THEN_the_ingredient_list_is_updated(){
+        //Given
+        let sandwichManager = SandwichManager(with: .xBacon)
+
+        //When
+        sandwichManager.addIngredient(.egg)
+
+        let xBacon = SandwichType.xBacon
+        var expectedResult = xBacon.ingredientsInSandwich
+        expectedResult.append(.egg)
```

Comment:
This seems to not be `when`, but the arrangement phase. I would suggest to move it to before the `sandwichManager.addIngredient(.egg)` 💅

Reply:
I applied that change, can you check again?